src/main/java/org/isf/medstockmovtype/manager/MedicaldsrstockmovTypeBrowserManager.java

```diff
@@ -58,28 +58,28 @@ protected void validateMovementType(MovementType movementType, boolean insert) t
 		List<OHExceptionMessage> errors = new ArrayList<>();
 		if (key.isEmpty()) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.medstockmovtype.pleaseinsertacode"),
+					MessageBundle.getMessage("angal.common.pleaseinsertacode.msg"),
 					OHSeverityLevel.ERROR));
 		}
 		if (key.length() > 10) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.medstockmovtype.codetoolongmaxchar"),
+					MessageBundle.getMessage("angal.common.thecodeistoolongmax10chars.msg"),
 					OHSeverityLevel.ERROR));
 		}
 		if (key2.length() > 2) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.medstockmovtype.typetoolongmaxchars"),
+					MessageBundle.getMessage("angal.medstockmovtype.thetypeistoolongmax2chars.msg"),
 					OHSeverityLevel.ERROR));
 		}
```

Comment:
should we use a format mask here?

Reply:
Did the first one but not the second because it is only used once.